docs/submission/submitting-molecular-data.md

```diff
@@ -134,9 +134,36 @@ If the file(s) successfully upload, then you will receive an `Upload completed`
 The final step to submitting molecular data is to set the state of an analysis to `PUBLISHED`. A published analysis signals to the DCC that this data is ready to be processed.
 
 ```shell
-> ./bin/sing publish -a a4142a01-1274-45b4-942a-01127465b422
+> bin/sing publish -a a4142a01-1274-45b4-942a-01127465b422
 
 AnalysisId a4142a01-1274-45b4-942a-01127465b422 successfully published
 ```
 
 Once your `sequencing_experiment` analysis has been successfully submitted and published, it will be queued for data processing. You can follow the progress of [molecular data processing](/docs/analysis-workflows/analysis-overview) for submitted data on your [Program Dashboard](/docs/submission/submitted-data).
+
+
+### Troubleshooting help
+
+
+#### Upload error
+
+During upload, a temporary file is written to the directory where the file that is being uploaded is located.  If you do not have permission to write to this directory, the upload will fail.  To address this, update the score-client `conf/application.properties` configuration file with a `client.uploadStateDir` parameter.
+
+```yaml title="score-client config"

```

Comment:
need to put the first line of content beneath the ``` line or it wont get shown.

Reply:
i dont think so - this isnt content.  its formatting and the title of the block

Reply:
yo this is a title to the code block. i've actually never seen this before, very cool.